### ISSUE TITLE
Create product vision

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#49]
## User story
1. As a Team
2. I want a good product vision
3. So that we have a long term goal

## Acceptance criteria
* [ ] Upload product vision to planning docs

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
